src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -1499,6 +1499,37 @@ class ScheduleLogger {
         }
     }
 
+    void Schedule::applyWellProdIndexScaling(const std::string& well_name, const std::size_t reportStep, const double scalingFactor) {
+        auto wstat = this->wells_static.find(well_name);
+        if (wstat == this->wells_static.end())
+            return;
+
+        auto uwell = wstat->second.unique();
```

Comment:
SInce this is borderline complex the `uwell` variable could maybe be `unique_wells`?

Reply:
> Since this is borderline complex the `uwell` variable could maybe be `unique_wells`?

Good idea.  I'll update accordingly.

Reply:
> Since this is borderline complex the `uwell` variable could maybe be `unique_wells`?

I've renamed the variable to `unique_well_instances`.